RussianFilter/sections/specific.txt

```diff
@@ -2628,7 +2628,7 @@ steaklovers.menu##.bannerb
 ||vkurier.by/wp-content/uploads/*/240_400.
 tlgg.ru##div[id^="ads"]
 serfery.ru,loxotrona.net##.b-adds
-ruspornovideo.su###secondary.widget-area
+alexgyver.ru,ruspornovideo.su###secondary.widget-area
```

Comment:
too wide rule, it blocks useful elements
f.e. here https://alexgyver.ru/gyverpower/

Reply:
let's add this domain to:
`edurusnews.ru##.widget_execphp`